### PR TITLE
Calculate your child maintenance: Replacing multiple choice YES/NO with Checkbox options

### PR DESCRIPTION
**Pull request by ikennaokpala**
[Trello card](https://trello.com/c/PornSWgk/700-5-uk-benefits-abroad-turning-yes-no-into-checkboxes)

## Motivation

As part of the accessibility improvements, we have been asked to replace the Yes/No multiple choice with checkboxes that contain the list of state benefits.

The intention here, is to assist devices like screen reader etc, to better recognise the checkbox values.

This PR content changes to the title and hints in the affected questions.

The question title has also changes as instructed in [1].

These changes have been requested via this document [1] by the content
design team.

[1] https://docs.google.com/document/d/1L7iCySnGFcTxTM8iyDJuYYRwSFT7nbNmwAOIgbZ5CZ0/edit#

## Factcheck

[Preview link](https://smart-answers-preview-pr-3180.herokuapp.com/calculate-your-child-maintenance/y/pay/1_child)
[GOVUK](https://gov.uk/calculate-your-child-maintenance/y/pay/1_child)

## Expected changes

- [Question 2](https://smart-answers-preview-pr-3180.herokuapp.com/calculate-your-child-maintenance/y/pay/1_child):
  - Replace Yes/No with list of benefits
  - Content update

### Before

screen shot 2017-08-04 at 16 38 26

### After
screen shot 2017-08-04 at 16 38 13

### CODE REVIEW

File: lib/smart_answer/calculators/child_maintenance_calculator.rb [hunk 1]
```diff
@@ -34,11 +34,8 @@ def rate_type
     end
 
     def calculate_maintenance_payment
-      if @benefits == 'no'
-        send("calculate_#{rate_type}_rate_payment")
-      else
-        0 #irrelevant what we return, with benefits rate is either nil or flat
-      end
+      return 0 if state_benefits?
```
Comment (ikennaokpala) [inline lib/smart_answer/calculators/child_maintenance_calculator.rb]: True.. wanted to keep relatively the same.

File: lib/smart_answer/calculators/child_maintenance_calculator.rb [hunk 2]
```diff
@@ -173,5 +170,39 @@ def total_yearly_fees(collect_fees)
     def self.child_maintenance_data
       @child_maintenance_data ||= YAML.load_file(Rails.root.join("lib/data/child_maintenance_data.yml"))
     end
+
+    def state_benefits?
+      benefits.present? && benefits.is_a?(Array) && valid_state_benefits?
```
Comment (ikennaokpala) [inline lib/smart_answer/calculators/child_maintenance_calculator.rb]: As I went through the possible unit test edge cases especially with the an empty or invalid string getting passed to benefits I believe that is where I added the `.present?`. At this point happy to remove `.present?`

File: lib/smart_answer_flows/calculate-your-child-maintenance/questions/gets_benefits.govspeak.erb [hunk 3]
```diff
@@ -7,8 +7,27 @@
 <% end %>
 
 <% options(
-  "yes": "Yes",
-  "no": "No"
+  "income_support": "Income Support",
```
Comment (ikennaokpala) [inline lib/smart_answer_flows/calculate-your-child-maintenance/questions/gets_benefits.govspeak.erb]: That is true I opted with going with this option.. it was easier to track the changes from regression test artefacts. 
At this point it may become some what relevant to have them defined in the calculator for this smart answer.

File: lib/smart_answer/calculators/child_maintenance_calculator.rb (hunk 1 above)
Comment (alecgibson) [inline lib/smart_answer/calculators/child_maintenance_calculator.rb]: Then can you at least include the comment? Otherwise it looks like `0` is an important value, when actually it isn't.

File: lib/smart_answer/calculators/child_maintenance_calculator.rb (hunk 2 above)
Comment (alecgibson) [inline lib/smart_answer/calculators/child_maintenance_calculator.rb]: Yes, please.

File: lib/smart_answer/calculators/child_maintenance_calculator.rb (hunk 1 above)
Comment (ikennaokpala) [inline lib/smart_answer/calculators/child_maintenance_calculator.rb]: Done
